Tasks/GradleV2/gradletask.ts

```diff
@@ -19,7 +19,8 @@ import ccUtil = require('codecoverage-tools/codecoverageutilities');
 import javacommons = require('java-common/java-common');
 import systemToken = require('utility-common/accesstoken');
 
-const accessTokenEnvSetting: string = 'VSTS_ENV_ACCESS_TOKEN';
```

Comment:
Put a comment here stating why we need to add to both and one needs to go away in next major task version.